R/constants.R

```diff
@@ -139,14 +148,45 @@ aglu.NO_AGLU_REGIONS <- "Taiwan"
 # Define GCAM category name of fertilizer
 aglu.FERT_NAME <- "N fertilizer"
 
+# Average Wood Density kg/m^3 for mass conversion
+aglu.AVG_WOOD_DENSITY_KGM3 <- 500
+
+# Average Agriculture Density for mass conversion
+# no idea of units, just hardcoded in as 1 in old DS
+aglu.AVG_AG_DENSITY <- 1
+
+# Forest Harvest Index
+aglu.FOREST_HARVEST_INDEX <- 0.8
+
+# Forest Erosion Control in kg/m^2
+aglu.FOREST_EROSION_CTRL_KGM2 <- 0.2
+
+#Mill Erosion Control in kg/m^2
+aglu.MILL_EROSION_CTRL_KGM2 <- 0
+
+# Wood energy content in GJ/kg
+aglu.WOOD_ENERGY_CONTENT_GJKG <- 0.0189
+
+# wood water content
+aglu.WOOD_WATER_CONTENT <- 0.065
+
+
+
 # XML-related constants
 aglu.GLU_NDIGITS          <- 3    # number of digits in the geographic land unit identifier codes
 aglu.GLU_NAME_DELIMITER   <- ""   # delimiter between the GLU name and number
+# ^ looks like this is in here twice
```

Comment:
Thanks for catching. I don't know that we need this comment anymore though since you have fixed it.